ogcapi-features/extensions/transaction/README.md

```diff
@@ -24,14 +24,63 @@ OGC API [Simple Transactions](http://docs.opengeospatial.org/DRAFTS/20-002.html)
+| `PUT /collections/{collectionId}/items/{featureId}`    | `application/json`  | partial Item                           | 200, 202, 204  | Updates an existing item by ID using a complete item description. |
+| `PATCH /collections/{collectionId}/items/{featureId}`  | `application/json`  | partial Item                           | 200, 202, 204  | Updates an existing item by ID using a partial item description.  |
+| `DELETE /collections/{collectionID}/items/{featureId}` | n/a                 | n/a                                    | 200, 202, 204  | Deletes an existing item by ID.                                   |
+
+### POST
+
+When the body is a partial Item:
+
+- Shall only create a new resource.
+- Shall return 409 if an Item exists for the same collection and id values.
+- Shall populate the `collection` field in the Item from the URI.
+- Shall return 201 and a Location header with the URI of the newly added resource for a successful operation.
+- May return the content of the newly added resource for a successful operation.
+
+When the body is a partial ItemCollection:
+
+- Shall only create a new resource.
+- Shall return 409 if an Item exists for any of the same collection and id values.
+- Shall populate the `collection` field in each Item from the URI.
+- Shall return 201 without a Location header.
+- May create only some of the Items in the ItemCollection.
+
+All cases:
+
+- Shall return 202 if the operation is queued for asynchronous execution.
+
+### PUT
+
+- Shall populate the `id` and `collection` fields in the Item from the URI.
+- Shall return 200 or 204 for a successful operation.
+- If 200 status code is returned, the server shall return the content of the updated resource for a successful operation.
+- Shall return 202 if the operation is queued for asynchronous execution.
+- Shall return 404 if no Item exists for this resource URI.
+- If the `id` or `collection` fields are different from those in the URI, status code 400 shall be returned.
+ 
+### PATCH
+
+- Shall populate the `id` and `collection` fields in the Item from the URI.
+- Shall return 200 or 204 for a successful operation.
+- If status code 200 is returned, the server shall return the content of the updated resource for a successful operation.
+- May return the content of the updated resource for a successful operation.
+- Shall return 202 if the operation is queued for asynchronous execution.
+- Shall return 404 if no Item exists for this resource URI.
+- If the `id` or `collection` fields are different from those in the URI, status code 400 shall be returned.
+
+PATCH is compliant with [RFC 7386](https://tools.ietf.org/html/rfc7386).
+
+### DELETE
+
+- Shall return 200 or 204 for a successful operation.
+- Shall return a 202 if the operation is queued for asynchronous execution.
```

Comment:
If it's queued it's not necessarily successful? Is this the best response you can give with asynchronous execution?

Reply:
This comes from OAFeat Part 4 -- yes, it's "successfully queued" rather than actually successful, since that's the best you can do.